core/backend/Process/LegacyHandler/EnablePortalUserActionHandler.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * SuiteCRM is a customer relationship management program developed by SalesAgility Ltd.
+ * Copyright (C) 2021 SalesAgility Ltd.
```

Comment:
Hi @eojedapilchik the copyright here should be 2023

Reply:
updated